pkg/client/clientset_generated/internalclientset/typed/core/unversioned/fake/fake_node_expansion.go

```diff
@@ -23,7 +23,7 @@ import (
 
 func (c *FakeNodes) PatchStatus(nodeName string, data []byte) (*api.Node, error) {
 	obj, err := c.Fake.Invokes(
-		core.NewPatchSubresourceAction(nodesResource, "status"), &api.Node{})
```

Comment:
How was this line generated before? I don't see any manual change that adds nodename and data to the calling of NewPatchSubresourceAction.